nixos/modules/services/torrent/deluge.nix

```diff
@@ -5,16 +5,19 @@ with lib;
 let
   cfg = config.services.deluge;
   cfg_web = config.services.deluge.web;
+  openFiles = 1024;
 in {
   options = {
     services.deluge = {
-      enable = mkOption {
-        default = false;
-        example = true;
+      enable = mkEnableOption "Start deluge daemon";
+
+      openFiles = mkOption {
```

Comment:
I think `maxOpenFiles` or `openFilesLimit` would be more clear naming or this variable.